production/loki-mixin/dashboards/loki-operational.libsonnet

```diff
@@ -13,11 +13,24 @@ local utils = import 'mixin-utils/utils.libsonnet';
                                showMultiCluster:: true,
                                clusterLabel:: $._config.per_cluster_label,
 
-                               matchers:: {
+                               hiddenRows:: [
+                                 'Cassandra',
```

Comment:
should we elevate another top level `$._config` value for toggling cassandra? I agree we don't need it for our cloud dashboards or the compiled mixin, but curious if there's no way to turn it on then there's no reason to keep it IMO.

Reply:
I think it makes sense to keep it for users that may be running cassandra.
I don't think we should add a new field to `$._config` for toggling this as only this dashboard is affected, and therefore, IMO it makes sense to add the config on the dashboard itself along with the rest of the dashboard-specific configs ([`showAnnotations` for example](https://github.com/grafana/loki/blob/a5ffb57004d827228d8f2a52bf2fd702d6fe976f/production/loki-mixin/dashboards/loki-operational.libsonnet#L11)).

Reply:
sounds good :+1: